### DISCUSSION TITLE
would Next version of Next-auth implement decouple CSRF-state ?

### DISCUSSION THREAD

**Discussion by isggwp** [nextauthjs/next-auth#3408] [Help]
### Question 💬

Ref to [this merge and review](https://github.com/nextauthjs/next-auth/pull/3142) its possible next version of next-auth implement decouple CSRF-state  by default?

### How to reproduce ☕️

just ask

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
not sure I understand. could you explain? that PR has been merged already.
